app/controllers/v3/service_credential_bindings_controller.rb

```diff
@@ -60,10 +60,12 @@ def create
     when 'app'
       app = get_app!(message.app_guid)
       unauthorized! unless can_create_and_destroy_service_bindings?(app.space)
+      suspended! unless permission_queryer.is_space_org_active?(app.space.guid)
```

Comment:
Add tests to spec/request/service_credential_bindings_spec.rb:
- post '/v3/service_credential_bindings' (app / key)
- patch '/v3/service_credential_bindings/:guid'
- delete '/v3/service_credential_bindings/:guid'